### PR TITLE
Resolve lint issues in "Working with the Catalog"

### PR DESCRIPTION
**Pull request by fiskus**

### CODE REVIEW

File: docs/walkthrough/working-with-the-catalog.md [hunk 1]
```diff
@@ -1,35 +1,44 @@
...
-The Quilt Catalog is the second half of Quilt. It provides an interface on top of your S3 bucket that brings Quilt features like data packages and search to the web.
+<!-- markdownlint-disable-next-line first-line-h1 -->
+The Quilt Catalog is the second half of Quilt. It provides an interface on top
+of your S3 bucket that brings Quilt features like data packages and search to
+the web.
 
 **[For a hands-on demo, check out the public demo catalog](https://open.quiltdata.com/b/quilt-example).**
 
-Note that you can use the Quilt Python API without using the catalog product, but they are designed to work together.
+Note that you can use the Quilt Python API without using the catalog product,
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:8]:
```suggestion
Note that you can use the Quilt Python API without using the Quilt Catalog,
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 2]
```diff
@@ -1,35 +1,44 @@
...
+but they are designed to work together.
 
 ## Brief tour
 
-The Quilt catalog provides a homepage for your catalog, based on a `README.md` file that you can optionally create at the top of your bucket.
+The Quilt catalog provides a homepage for your catalog, based on a `README.md`
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:13]:
```suggestion
The Quilt Catalog provides a homepage for your data catalog, based on a `README.md`
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 3]
```diff
@@ -1,35 +1,44 @@
...
+file that you can optionally create at the top of your bucket.
 
 ### Browse
 
-![](../imgs/catalog_homepage.png)
+![Homepage](../imgs/catalog_homepage.png)
 
 The catalog lets you navigate packages in the registry on the packages tab.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:20]:
```suggestion
The Catalog lets you navigate packages in the registry using the "Packages" tab.
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 4]
```diff
@@ -1,35 +1,44 @@
...
 
-![](../imgs/catalog_packages_tab.png)
+![Packages tab](../imgs/catalog_packages_tab.png)
 
 You can also browse the underlying S3 files using the files tab.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:24]:
```suggestion
You can also browse the underlying S3 files using the "Bucket" tab.
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 5]
```diff
@@ -1,35 +1,44 @@
...
 
-![](../imgs/catalog_files_tab.png)
+![Files tab](../imgs/catalog_files_tab.png)
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:26]: Needs to be replaced with a new screenshot - see previous comments

File: docs/walkthrough/working-with-the-catalog.md [hunk 6]
```diff
@@ -1,35 +1,44 @@
...
 ## Brief tour
 
-The Quilt catalog provides a homepage for your catalog, based on a `README.md` file that you can optionally create at the top of your bucket.
+The Quilt catalog provides a homepage for your catalog, based on a `README.md`
+file that you can optionally create at the top of your bucket.
 
 ### Browse
 
-![](../imgs/catalog_homepage.png)
+![Homepage](../imgs/catalog_homepage.png)
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:18]: Needs to be replaced with a new screenshot - we don't have a "Files" tab anymore, it's been replaced with "Bucket"

File: docs/walkthrough/working-with-the-catalog.md [hunk 7]
```diff
@@ -1,35 +1,44 @@
...
 
-Catalogs also enable you to search the contents of your bucket. We support both unstructured (e.g. "`San Francisco`") and structured with [Query String Queries](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/query-dsl-query-string-query.html#query-string-syntax) (e.g. "`metadata_key: metadata_value`") search. Hits are previewed right in the search results.
+Catalogs also enable you to search the contents of your bucket. We support both
+unstructured (e.g. "`San Francisco`") and structured with
+[Query String Queries](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/query-dsl-query-string-query.html#query-string-syntax)
+(e.g. "`metadata_key: metadata_value`") search. Hits are previewed right in the
+search results.
 
-![](../imgs/catalog_search.png)
+![Search](../imgs/catalog_search.png)
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:36]: Needs to be replaced with a new screenshot - see previous comments

File: docs/walkthrough/working-with-the-catalog.md [hunk 8]
```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
...
+feature consult [workflows](../advanced-features/workflows.md) page.
 
 ### Summarize
 
-Adding a `quilt_summarize.json` file to a data package (or S3 directory path) will enable content preview right on the landing page.
+Adding a `quilt_summarize.json` file to a data package (or S3 directory path)
+will enable content preview right on the landing page.
 
-![](../imgs/catalog_package_landing_page.png)
+![Package landing page](../imgs/catalog_package_landing_page.png)
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:76]: Needs to be replaced with a new screenshot - see previous comments

File: docs/walkthrough/working-with-the-catalog.md [hunk 9]
```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
...
 
-You can push the existing package from one bucket to another. To use this feature consult [workflows](../advanced-features/workflows.md) page.
+You can push the existing package from one bucket to another. To use this
+feature consult [workflows](../advanced-features/workflows.md) page.
 
 ### Summarize
 
-Adding a `quilt_summarize.json` file to a data package (or S3 directory path) will enable content preview right on the landing page.
+Adding a `quilt_summarize.json` file to a data package (or S3 directory path)
+will enable content preview right on the landing page.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:74]: Add a link to catalog/visualizationdashboards#quilt_summarize.json which is a more comprehensive overview.

File: docs/walkthrough/working-with-the-catalog.md [hunk 10]
```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
...
 
-![](../imgs/catalog_package_landing_page.png)
+![Package landing page](../imgs/catalog_package_landing_page.png)
 
-Colocating data with context in this way is a simple way of making your data projects approachable and accessible to collaborators.
+Colocating data with context in this way is a simple way of making your data
+projects approachable and accessible to collaborators.
 
-`quilt_summarize.json` can be a list of paths to files in S3 that you want to include in your summary. For example: `["description.md", "../notebooks/exploration.ipynb"]`. Additionally, note that if a `README.md` file is present, it will always be rendered as well.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:64]: Remove all of this - duplicate content to other page linked above.

File: docs/walkthrough/working-with-the-catalog.md [hunk 11]
```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
...
+`quilt_summarize.json` can be a list of paths to files in S3 that you want to
+include in your summary. For example:
+`["description.md", "../notebooks/exploration.ipynb"]`. Additionally, note that
+if a `README.md` file is present, it will always be rendered as well.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:84]: Remove

File: docs/walkthrough/working-with-the-catalog.md [hunk 12]
```diff
@@ -39,39 +48,52 @@ Files are the content of your package.
...
 
 > There are currently some small limitations with preview:
 >
-> * Objects linked to in `quilt_summarize.json` are always previewed as of the latest version, even if you are browsing an old version of a package.
-> * Object titles and image thumbnails link to the file view, even if you are in the package view.
+> * Objects linked to in `quilt_summarize.json` are always previewed as of the
+>   latest version, even if you are browsing an old version of a package.
+> * Object titles and image thumbnails link to the file view, even if you are in
+>   the package view.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:91]: Is this still true? If so, it should be moved to catalog/visualizationdashboards#quilt_summarize.json and deleted from this page.

**Review by robnewman — CHANGES_REQUESTED** [review]

File: docs/walkthrough/working-with-the-catalog.md (hunk 12 above)
Comment (fiskus) [reply docs/walkthrough/working-with-the-catalog.md]: Yes, it's true

I moved it to Visualizations and Dashboards page

File: docs/walkthrough/working-with-the-catalog.md [hunk 13]
```diff
@@ -1,35 +1,44 @@
...
+![Homepage](../imgs/catalog_homepage.png)
 
-The catalog lets you navigate packages in the registry on the packages tab.
+The Catalog lets you navigate packages in the registry using the "Packages" tab.
 
-![](../imgs/catalog_packages_tab.png)
+![Packages tab](../imgs/catalog_packages_tab.png)
 
-You can also browse the underlying S3 files using the files tab.
+You can also browse the underlying S3 files using the "Bucket" tab.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:24]:
```suggestion
You can also browse the underlying S3 objects using the "Bucket" tab.
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 14]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
 
 The associated workflow contains the rules for validating your package.
 
-The metadata can be added with JSON editor for both packages and individual file entries within a package, represented as a key/value table with infinite nesting. If workflow contains JSON schema, you will have predefined key/value pairs according to the schema.
+The metadata can be added with JSON editor for both packages and individual file
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:51]:
```suggestion
The metadata can be added with the JSON editor for both packages and individual file
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 15]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
+entries within a package, represented as a key/value table with infinite
+nesting. If workflow contains JSON schema, you will have predefined key/value
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:53]:
```suggestion
nesting. If your workflow contains a JSON schema, you will have predefined key/value
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 16]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
+pairs according to the schema.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:54]:
```suggestion
pairs based on the schema.
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 17]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
 
 #### JSON editor
 
-To add a new key/value field double click on an empty cell and type key name, then press "Enter" or "Tab", or click outside of the cell. To change value double click on that value.
+To add a new key/value field double click on an empty cell and type key name,
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:58]:
```suggestion
To add a new key/value field double click on an empty cell and type the key name,
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 18]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
+then press "Enter" or "Tab", or click outside of the cell. To change value
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:59]:
```suggestion
then press "Enter" or "Tab", or click outside of the cell. To change the value
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 19]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
+double click on that value.
 
-Values can be strings, numbers, arrays, or objects. Every value that you type will be parsed as JSON.
+Values can be strings, numbers, arrays, or objects. Every value that you type
+will be parsed as JSON.
 
 We don't support references and compound types yet.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:65]:
```suggestion
> Limitations
> * References and compound types are not currently supported.
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 20]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
 
 ### Push to bucket
 
-You can push the existing package from one bucket to another. To use this feature consult [workflows](../advanced-features/workflows.md) page.
+You can push the existing package from one bucket to another. To use this
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:69]:
```suggestion
You can push an existing data package from one S3 bucket to another. To use this
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 21]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
-> There are currently some small limitations with preview:
->
-> * Objects linked to in `quilt_summarize.json` are always previewed as of the latest version, even if you are browsing an old version of a package.
-> * Object titles and image thumbnails link to the file view, even if you are in the package view.
+![Package landing page](../imgs/catalog_package_landing_page.png)
 
 ## Admin UI
 
-The Quilt catalog includes an admin panel that allows you to manage users and buckets in your stack and to customize your Quilt catalog.
+The Quilt catalog includes an admin panel that allows you to manage users and
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:84]:
```suggestion
The Quilt Catalog includes an Admin panel where you can manage users and
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 22]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
+buckets in your stack and to customize your Quilt catalog.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:85]:
```suggestion
buckets in your stack and customize the display of your Quilt Catalog.
```

**Review by robnewman — CHANGES_REQUESTED** [review]
Minor grammar and typos

File: docs/walkthrough/working-with-the-catalog.md [hunk 23]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
+Values can be strings, numbers, arrays, or objects. Every value that you type
+will be parsed as JSON.
 
 We don't support references and compound types yet.
 
 ### Push to bucket
 
-You can push the existing package from one bucket to another. To use this feature consult [workflows](../advanced-features/workflows.md) page.
+You can push the existing package from one bucket to another. To use this
+feature consult [workflows](../advanced-features/workflows.md) page.
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:72]:
```suggestion
feature consult the [Workflows](../advanced-features/workflows.md) page.
```

File: docs/walkthrough/working-with-the-catalog.md [hunk 24]
```diff
@@ -39,39 +48,41 @@ Files are the content of your package.
...
 
 ### Summarize
 
-Adding a `quilt_summarize.json` file to a data package (or S3 directory path) will enable content preview right on the landing page.
+Adding a `quilt_summarize.json` file to a data package (or S3 directory path)
+will enable content preview right on the landing page.
 
-![](../imgs/catalog_package_landing_page.png)
+See [Visualization & dashboards docs](../Catalog/VisualizationDashboards.md#quilt_summarize.json)
```
Comment (robnewman) [inline docs/walkthrough/working-with-the-catalog.md:79]:
```suggestion
See [Visualization & dashboards documentation](../Catalog/VisualizationDashboards.md#quilt_summarize.json)
```

**Review by robnewman — CHANGES_REQUESTED** [review]
Many suggested grammar changes in `working-with-the-catalog.md` were marked `resolved` but the changes weren't applied? Did you decide they weren't relevant?

**Pull request merged**